boredgamers/__main__.py

```diff
@@ -0,0 +1,39 @@
+from boredgamers import db, app
+from boredgamers.models import User, Game
+import argparse
+import csv
+
+
+# parse arguments:
+#parser = ArgumentParser()
+#parser.add_argument("")
+#args = parser.parse_args()
+
+
+# create all tables (if they don't exist):
+db.create_all()
```

Comment:
We probably want to execute this based on a specific sub-command. For example

    python3 -m boredgamers init_db

You could look into [sub-commands](https://docs.python.org/3/library/argparse.html#sub-commands) for this.

Then have a sub-command for database initialization, a sub-command for starting the app server, and so on.